### PR TITLE
Add subcategory in documentation page

### PR DESCRIPTION
**Pull request by yomovh**
# Description
In order to group the resources / data source of the documentation page on the registry of Hashicorp : https://registry.terraform.io/providers/ovh/ovh/latest/docs

I have added a [subcategory](https://developer.hashicorp.com/terraform/registry/providers/docs#subcategories) in the YAMLFrontmatter of the documentation files. I have also removed the other attributes which were not useful and not  rendered in the Terraform Registry web UI (see [here](https://developer.hashicorp.com/terraform/registry/providers/docs#yaml-frontmatter))

The category naming are the official ones, courtesy of the product map repository https://github.com/baaastijn/ovh_product_map

## Type of change

- [x] Documentation update

# How Has This Been Tested?
Not applicable

# Checklist:
Not applicable

### CODE REVIEW

**Comment by yomovh** [comment]
Done ;-)

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-check#1532.0 ✔ 
* 
  * checks ✔

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-check#1533.0 ✔ 
* 
  * checks ✔

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-testacc#1532.0 ✘  
* 
  * checks ✘

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-testacc#1533.0 ✘  
* 
  * checks ✘

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ This project accepts contributions. In order to contribute, you should pay atten
 - Code must be formatted with `make fmt` command
 - Name your resources and datasources according to the API endpoint
 - Check your documentation through [Terraform Doc Preview Tool](https://registry.terraform.io/tools/doc-preview)
+- When adding a documentation page, use the `subcategory:` tag in the [YAML Frontmatter](https://developer.hashicorp.com/terraform/registry/providers/docs#yaml-frontmatter) with a value equals to the product name defined in the [product map](https://www.product-map.ovh/)
```
Comment (scraly) [inline CONTRIBUTING.md:17]:
```suggestion
- When adding a documentation page, use the `subcategory:` tag in the [YAML Frontmatter](https://developer.hashicorp.com/terraform/registry/providers/docs#yaml-frontmatter) with a value equals to the product name defined in the OVHcloud [product map](https://www.product-map.ovh/)
```

**Review by scraly — CHANGES_REQUESTED** [review]

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-check#1536.0 ✔ 
* 
  * checks ✔

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-check#1535.0 ✔ 
* 
  * checks ✔

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-testacc#1536.0 ✘  
* 
  * checks ✘

**Review by scraly — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-testacc#1535.0 ✘  
* 
  * checks ✘

**Pull request merged**